docs/README.md

```diff
@@ -4,14 +4,15 @@ This page has instructions for collecting Kubernetes logs, metrics, and events;
 metadata; and sending them to Sumo Logic. See our [documentation guide](https://help.sumologic.com/docs/observability/kubernetes/) for
 details on our Kubernetes Solution.
 
-- [Solution overview](#solution-overview)
-  - [Log Collection](#log-collection)
-  - [Metrics Collection](#metrics-collection)
-  - [Kubernetes Events Collection](#kubernetes-events-collection)
-- [Minimum Requirements](#minimum-requirements)
-- [Support Matrix](#support-matrix)
-  - [ARM support](#arm-support)
-  - [Falco support](#falco-support)
+- [Deployment Guide for unreleased version](#deployment-guide-for-unreleased-version)
```

Comment:
The difference is we do not plan to add anything new to the the branch, otherwise we need to update `release.md` as well as this line will have to be updated in case of release

Reply:
Then we can make this change after we're actually not adding anything to the branch. Right now we're still doing it.